### PR TITLE
Update h1 tags across the training-kit

### PR DESCRIPTION
**Pull request by brianamarie**
This pull request addresses an internal action item to improve our website.

>  Improve `<h1>` tags (unique to page content) 

Please review for:
- Are the titles actually better and more descriptive? 
- Am I missing any pages?
- Are there duplicate titles where we can make them all different?

cc @github/services-training @github/services-operations @leereilly @BonnieC

### CODE REVIEW

**Comment by crichID** [comment]
@brianamarie do you think we should resolve all of these merge conflicts so we can do a :sparkles: review?

**Comment by brianamarie** [comment]
@crichID that's a fantastic point! Merge conflicts have been resolved. 😄

File: _data/navigation.yml [hunk 1]
```diff
@@ -10,39 +10,39 @@ main:
...
+  - title: "Explore the GitHub Repository"
     url: /intro-to-github/03/
   - title: "Use GitHub Flow"
     url: /intro-to-github/04/
   - title: "Create a Branch"
     url: /intro-to-github/05/
   - title: "Find Your Coordinates"
     url: /intro-to-github/06/
-  - title: "Work on GitHub"
+  - title: "Working on GitHub"
```
Comment (crichID) [inline _data/navigation.yml]: @brianamarie I know this is a picky thing, but I wonder if we should try to maintain consistent structure in our titles (e.g. should we use -ing or not)?

File: _data/navigation.yml [hunk 2]
```diff
@@ -10,39 +10,39 @@ main:
...
     url: /intro-to-github/02/
-  - title: "Explore the Repository"
+  - title: "Explore the GitHub Repository"
     url: /intro-to-github/03/
   - title: "Use GitHub Flow"
     url: /intro-to-github/04/
   - title: "Create a Branch"
     url: /intro-to-github/05/
   - title: "Find Your Coordinates"
     url: /intro-to-github/06/
```
Comment (crichID) [inline _data/navigation.yml]: @brianamarie did the SEO experts also make the point that we should use more than just 06/ for our perma links? I believe they were saying our URLs should also be descriptive??

File: _data/navigation.yml (hunk 1 above)
Comment (brianamarie) [inline _data/navigation.yml]: @crichID Not picky at all, very good point! Which should we use? To 'ing' or not to 'ing'? 🤔

File: _data/navigation.yml (hunk 2 above)
Comment (brianamarie) [inline _data/navigation.yml]: @crichID Great point!!! Commit incoming. 👍

**Comment by brianamarie** [comment]
As I look through the `_data/navigation.yml`, I see a [big chunk for some patchwork courses](https://github.com/github/training-kit/pull/471/files#diff-b3b881748eca032c02d212a5ba258a9bR48), but I'm not able to find the files for the course anywhere. @github/services-training  Sanity check...did this course used to exist, but we deleted it for some reason?

File: _data/navigation.yml (hunk 2 above)
Comment (brianamarie) [inline _data/navigation.yml]: @crichID last 4 commits address this point

**Comment by hectorsector** [comment]
> did this course used to exist, but we deleted it for some reason

@brianamarie if I remember correctly, it's how we began to think about a GH Pages offering but we eventually opted for a general public course that Patchwork could _also_ use? I remember @beardofedu having something to do with this. Also, this is related:

- https://github.com/github/on-demand/pull/89
- [Source repo `blame` view](https://github.com/github/on-demand/blame/gh-pages/_data/navigation.yml#L36)

**Comment by crichID** [comment]
@brianamarie I changed the three pages that were using -ing since we were showing a bit of a preference toward -ing less titles :wink: I also removed the patchwork course from the nav. I will mark the changes as approved but I would :heart: one more quick set of :eyes: from @github/services-training.

**Comment by brianamarie** [comment]
@crichID I spun this up locally and it looks great!

There was an odd error unrelated to the changes you made about a mapping value due to a colon in a title, but I changed that. 

@crichID 🙇 Thank you for your review. Again, pinging @github/services-training about this change since it will affect how we edit existing courses and how we create them in the future.

**Pull request merged**